statsmodels/tsa/filters/hp_filter.py

```diff
@@ -18,6 +18,12 @@ def hpfilter(x, lamb=1600):
         suggested for quarterly data. Ravn and Uhlig suggest using a value
         of 6.25 (1600/4**4) for annual data and 129600 (1600*3**4) for monthly
         data.
+    two_sided : bool, optional
```

Comment:
Maybe `one_sided=False`.  This makes it clearer that the "default" or "usual" case is 2-sdied.